src/types/gtm.d.ts

```diff
@@ -0,0 +1,17 @@
+declare global {
+  interface Window {
+    dataLayer?: object[]
+  }
+}
+
+interface OfferData {
+  insurance_type: TypeOfContract
+  campaign_code: 'yes' | 'no'
+  number_of_people: number
+  insurance_price: string
+}
+
+export interface DataLayerObject {
+  event: string
+  offerData?: UserData
```

Comment:
```suggestion
  offerData?: OfferData
``` 
Right?

Reply:
Ah, thanks for catching that one! 🙌 Weird that typescript didn't complain on a missing type 🤔